src/components/navigationBar/NavigationBar.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from "react";
+import { getSharedUserSample } from "../util/api";
+import Header from "./Header";
+import LinkLogo from "./LinkLogo";
+import UserProfileImg from "./UserProfileImg";
+import StyledLink from "./StyledLink";
+import UserProfileContainer from "./UserProfileContainer";
+import StyledLoginLink from "./StyledLoginLink";
+import { useLocation } from "react-router-dom";
+
+function NavigationBar() {
+  const location = useLocation();
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  const [userProfile, setUserProfile] = useState(null);
+
+  const handleSharedUserProfile = async () => {
+    const result = await getSharedUserSample();
+    if (!result) return;
+
+    const { email, image_source } = result.data[0];
+    setUserProfile({ email, image_source });
+  };
+
+  const isSticky = () => {
+    if (location.pathname === "/folder" || location.pathname === "/")
+      return false;
+    return true;
+  };
```

Comment:
```suggestion
  const isSticky = () => location.pathname !== "/folder" && location.pathname !== "/"
```

header 가 stikcy 한지를 나타내는 변수이기 때문에 `isHeaderSticky` 정도의 네이밍 추천드려요!